{{cookiecutter.github_repository_name}}/{{cookiecutter.app_name}}/users/serializers.py

```diff
@@ -7,8 +7,8 @@ class UserSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = User
-        fields = ('id', 'username', 'auth_token', 'first_name', 'last_name',)
```

Comment:
ahh I see from the spit diff that you also kept the first `'` from `'auth_token'`. Merging this through. @glemmaPaul